README.md

```diff
@@ -21,8 +21,17 @@ is also available via
 
 ## Quick start guide
 
-To track request latency broken down by server name and request count broken
-down by server name and status, add the following to `nginx.conf`:
+In case you use an external module for nginx-lua-support (e.g. the
+`libnginx-mod-http-lua` package on Debian) first add
```

Comment:
As far as I can tell, this is not necessary. Debian package automatically adds a symlink to `/etc/nginx/modules-enabled` which enables the module (please see `/var/lib/dpkg/info/libnginx-mod-http-lua.postinst`).

Reply:
That's with `libnginx-mod-http-lua`. Have you looked at `/var/lib/dpkg/info/libnginx-mod-http-lua.postinst`?

Reply:
I wasn't aware of that. I'll update the description accordingly.